camera_to_quad_matrix: [-1, 0, 0, 0, -1, 0, 0, 0, -1]
camera_translation: [0.20, 0.0, -0.02]
platform_height: 0.45                                 #double: Height platform used
loop_rate: 10